indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -331,7 +331,9 @@ public static boolean runJobs(List<Jobby> jobs, HadoopDruidIndexerConfig config)
         Path workingPath = config.makeIntermediatePath();
         log.info("Deleting path[%s]", workingPath);
         try {
-          workingPath.getFileSystem(injectSystemProperties(new Configuration())).delete(workingPath, true);
+          if( workingPath.getFileSystem(injectSystemProperties(new Configuration())).delete(workingPath, true) ){
+            existing.clear();
```

Comment:
maybe I should rollback to the problematic version and see why some job would success.
I'll do that in the weekend